src/status_im2/contexts/profile/settings/header/view.cljs

```diff
@@ -0,0 +1,46 @@
+(ns status-im2.contexts.profile.settings.header.view
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im2.common.not-implemented :as not-implemented]
+            [status-im2.contexts.profile.settings.header.avatar :as header.avatar]
+            [status-im2.contexts.profile.settings.header.header-shape :as header.shape]
+            [status-im2.contexts.profile.settings.header.style :as style]
+            [status-im2.contexts.profile.utils :as profile.utils]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  [{:keys [theme scroll-y]}]
```

Comment:
This view is using `theme` without `with-theme` HOC. This would result in a nil value.

Reply:
thanks @smohamedjavid , resolved